features/track.yml

```diff
@@ -1,6 +1,5 @@
-draft_date: 2024-06-05
 name: <track>
```

Comment:
I think this feature should be named "Text tracks" to match the existing "Audio and video tracks" feature and include the generic (non-WebVTT) text track API as well. This is a good start, from `npm run traverse | grep TextTrack`:

api.HTMLMediaElement.addTextTrack
api.TextTrack
api.TextTrack.activeCues
api.TextTrack.addCue
api.TextTrack.cuechange_event
api.TextTrack.cues
api.TextTrack.id
api.TextTrack.inBandMetadataTrackDispatchType
api.TextTrack.kind
api.TextTrack.label
api.TextTrack.language
api.TextTrack.mode
api.TextTrack.removeCue
api.TextTrackCue
api.TextTrackCue.endTime
api.TextTrackCue.enter_event
api.TextTrackCue.exit_event
api.TextTrackCue.id
api.TextTrackCue.pauseOnExit
api.TextTrackCue.startTime
api.TextTrackCue.track
api.TextTrackCueList
api.TextTrackCueList.getCueById
api.TextTrackCueList.length
api.TextTrackList
api.TextTrackList.addtrack_event
api.TextTrackList.change_event
api.TextTrackList.getTrackById
api.TextTrackList.length
api.TextTrackList.removetrack_event

This feature would cover both external text tracks from `<track>` and in-band text tracks.

Reply:
done